<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BrowserAccessControlDialog</class>
 <widget class="QDialog" name="BrowserAccessControlDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KeePassXC - Browser Access Request</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="siteLabel">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>%1 is requesting access to the following entries:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="CustomTableWidget" name="itemsTable">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="cornerButtonEnabled">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="rememberDecisionCheckBox">
       <property name="toolTip">
        <string>Remember access to checked entries</string>
       </property>
       <property name="accessibleName">
        <string>Remember access to checked entries</string>
       </property>
       <property name="text">
        <string>Remember</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="allowButton">
       <property name="accessibleName">
        <string>Allow access to entries</string>
       </property>
       <property name="text">
        <string>Allow Selected</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="denyButton">
       <property name="text">
        <string>Deny All</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>rememberDecisionCheckBox</tabstop>
  <tabstop>allowButton</tabstop>
  <tabstop>denyButton</tabstop>
 </tabstops>
 <customwidgets>
  <customwidget>
   <class>CustomTableWidget</class>
   <extends>QTableWidget</extends>
   <header>browser/CustomTableWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
